content/realtime/authentication.textile

```diff
@@ -159,7 +159,7 @@ minimize. See example
   ```[csharp]
     AblyRealtime realtime = new AblyRealtime("{{API_KEY}}");
     TokenParams tokenParams = new TokenParams {ClientId = "Bob"};
-    TokenRequest tokenRequest = await realtime.Auth.CreateTokenRequestAsync(tokenParams);
+    String tokenRequest = await realtime.Auth.CreateTokenRequestAsync(tokenParams);
```

Comment:
lower case `string` is the preferred keyword (although `string` is actually and alias for `System.String` and as the `System` namespace is imported by default `String` is still valid)